src/ConfigurationsProvider.tsx

```diff
@@ -24,6 +39,7 @@ const mergeDefaultConfiguration = (target: any, source: any) => {
 
 const defaultConfiguration = {
   appDesignSystem: defaultAppDesignSystemConfig,
+  resourceFilters: defaultFilterConfig,
```

Comment:
`defaultFilterConfig` should also be renamed to `defaultResourceFilterConfig` as well as all other occurrences

Reply:
ah i see there are changes according to the names in the follow up pull request https://github.com/smart-village-solutions/smart-village-app-app/pull/1188

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a